tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -37,7 +37,7 @@ def setUpClass(cls):
         })
 
     @tier1
-    def test_create_cvf_different_names(self):
+    def test_positive_create_with_name_by_cv_id(self):
```

Comment:
imo `test_positive_create_with_name_by_cv_id` is shorter and still makes sence. Also this is the approach we use almost everywhere in tests names, so i'd vote for leaving it as it is. Same for `assign`.